docs/pages/9 - Contrib Modules.md

```diff
@@ -912,3 +912,62 @@ Publishing to custom local Maven repository
 [40/40] project.publishM2Local
 Publishing to /tmp/m2repo
 ```
+
+### Bloop
+
+This plugin generates [bloop](https://scalacenter.github.io/bloop/) configuration 
+from your build file, which lets you use the bloop CLI for compiling, and makes 
+your scala code editable in [Metals](https://scalameta.org/metals/)   
+
+
+#### Quickstart:
+```scala
+// build.sc (or any other .sc file it depends on, including predef) 
+// Don't forget to replace VERSION
+import $ivy.`com.lihaoyi::mill-contrib-bloop:VERSION`
```

Comment:
Is it possible to programmatically run this setup? For example `mill --extra-predef /tmp/metals.sc mill.contrib.Bloop/install`. It would be nice if we could support mill in the "import build" command in Metals, the same way it works for sbt.

Reply:
Just had a go at it : ` mill --predef /tmp/metals.sc mill.contrib.Bloop/install` would work

Reply:
Cool, we're working on adding automatic "import build" for Gradle in Metals, which requires abstracting some sbt-specifics away. I'm optimistic it won't be too much effort to add automatic "import build" for Mill after that.